### PR TITLE
fix(CSS): add :host, :host-context, and ::slotted selector examples

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- Adds examples for https://github.com/mdn/content/issues/27938

To test checkout the branch in local then run
```bash
npm install
npm run build
npm start
```

Open following URLs in browser:
```
http://localhost:9090/pages/tabbed/pseudo-class-host.html
http://localhost:9090/pages/tabbed/pseudo-class-host_function.html
http://localhost:9090/pages/tabbed/pseudo-class-host-context.html
http://localhost:9090/pages/tabbed/pseudo-element-slotted.html
```

### CODE REVIEW

File: live-examples/css-examples/pseudo-class/host-context.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* Following CSS is being put inside the shadow DOM. */
```
Comment (Rumyra) [inline live-examples/css-examples/pseudo-class/host-context.css]:
```suggestion
/* Following CSS is being applied inside the shadow DOM. */
```

File: live-examples/css-examples/pseudo-class/host.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/* This CSS is being put inside the shadow DOM. */
```
Comment (Rumyra) [inline live-examples/css-examples/pseudo-class/host.css]:
```suggestion
/* This CSS is being applied inside the shadow DOM. */
```

File: live-examples/css-examples/pseudo-class/host_function.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/* Following CSS is being put inside the shadow DOM. */
```
Comment (Rumyra) [inline live-examples/css-examples/pseudo-class/host_function.css]:
```suggestion
/* Following CSS is being applied inside the shadow DOM. */
```

File: live-examples/css-examples/pseudo-element/slotted.css [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/* This CSS is being put inside the shadow DOM. */
```
Comment (Rumyra) [inline live-examples/css-examples/pseudo-element/slotted.css]:
```suggestion
/* This CSS is being applied inside the shadow DOM. */
```

**Review by Rumyra — CHANGES_REQUESTED** [review]

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
Thanks @OnkarRuikar - these look great :D

**Comment by NiedziolkaMichal** [comment]
Too bad I created similar examples more than a year ago and they were never reviewed: https://github.com/mdn/interactive-examples/pull/2177 https://github.com/mdn/interactive-examples/pull/2186
I will close those PRs.

**Comment by OnkarRuikar** [comment]
> Too bad I created similar examples more than a year ago and they were never reviewed: #2177 #2186 I will close those PRs.

:cold_sweat: I should've looked for existing PRs first. Never though something that old would already exist.

**Comment by sideshowbarker** [comment]
> Too bad I created similar examples more than a year ago and they were never reviewed: https://github.com/mdn/interactive-examples/pull/2177 https://github.com/mdn/interactive-examples/pull/2186
> I will close those PRs.

Please don't close them. I have been reviewing and merging the open PRs here, and I plan to get all of them merged soon.

**Pull request merged**